js/package.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "jupyter-threejs",
-  "version": "0.1.3",
+  "version": "0.2.0",
   "description": "jupyter - threejs bridge",
   "author": "Jupyter-Threejs development team",
-  "license": "BSD",
+  "license": "BSD-3-Clause",
   "main": "src/index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/jovyan/pythreejs.git"
   },
   "scripts": {
     "clean": "node ./scripts/clean-generated-files.js",
+    "enums": "node ./scripts/generate-enums.js",
```

Comment:
Figured it was easier to keep it separate initially and then combine it later if wanted than the opposite. It also makes it easier to work on only one of them without having to spend time waiting for the autogen processing the other (unchanged) code. I'm very open for having a single npm command call both scripts though.